common/core/build.gradle.kts

```diff
@@ -26,12 +19,6 @@ kotlin {
         }
     }
     sourceSets {
-        all {
-            with(languageSettings) {
-                useExperimentalAnnotation("kotlin.RequiresOptIn")
-                useExperimentalAnnotation("kotlin.ExperimentalUnsignedTypes")
```

Comment:
We still use these annotation afaik. Perhaps unsigned types are no longer experimental, will take a look.

Reply:
got it